MathlibTest/TautoSet.lean

```diff
@@ -0,0 +1,151 @@
+/-
+Copyright (c) 2025 Lenny Taelman. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Lenny Taelman
+-/
+
+import Mathlib.Tactic.TautoSet
+
+variable (α : Type) (A B C D E : Set α)
```

Comment:
It doesn't matter for examples; however, I would make them all implicit nevertheless.